doc/manual/castxml.1.rst

```diff
@@ -99,3 +99,52 @@ case, CastXML always adds the following predefined macros:
 
 Source files may use these to identify the tool that is actually doing the
 preprocessing even when ``--castxml-cc-<id>`` changes the predefined macros.
+
+FAQ
+===
+
+Why are C++ function bodies not dumped in XML?
+----------------------------------------------
+
+The original sponsors of the project had no need for function bodies.
+So far the authors have not had time or funding to implement the support.
```

Comment:
I like the wording used for templates below better:


```rst
This feature has not been implemented because the driving project for which
CastXML was written had no need for function bodies.
```